Source/DafnyStandardLibraries/Makefile

```diff
@@ -8,7 +8,8 @@ DOO_FILE_TARGET=binaries/DafnyStandardLibraries.doo
 all: check-binary test check-format check-examples
 
 verify:
-	$(DAFNY) verify src/DafnyStdLibs/dfyconfig.toml
```

Comment:
Was this just missed in the `DafnyStdLibs` -> `Std` renaming?

Reply:
Yup, forgot to call it out in the PR description